app/app.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	AppVersion = "RC-0.11.1"
```

Comment:
This is a consensus breaking change, so we need to go with `0.12`

Reply:
Ah right. Changed it to RC-0.12.  Ideally we should do BETA, but let's cut that step this time.